wordpress/containerbuddy/onstart.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Refresh all config files in order
+#
+# This script is typically called once at the container start, but
+# it can be called manually if WP config details must be changed
+
+# The database and memcached config files are separate to avoid collisions
+# if their backends' onChange handlers are triggered simultaneously
+echo "******running onstart script*********"
+
+until [[ `curl -s consul:8500/v1/health/state/passing | grep mysql-primary` ]]
```

Comment:
I'm ok adding `jq` to images. It installs easy and doesn't bring along any dependencies.

If we have Node.js in the image, I also use [`json`](https://github.com/trentm/json) (install: `npm install -g json`) which is similar. We maintain `json` and use/ship it in [our software](https://github.com/joyent/sdc), so there's a family preference for it (but not if we're not using Node.js for other things already).